save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/TopBar.kt

```diff
@@ -109,10 +117,28 @@ fun topBar() = FC<TopBarProps> { props ->
                     .split(URL_PATH_DELIMITER)
                     .filterNot { it.isBlank() }
                     .apply {
-                        foldIndexed("#") { index: Int, acc: String, pathPart: String ->
-
-                            val currentLink = "$acc/$pathPart"
-
+                        val insideTab = location.pathname.substringBeforeLast("?").let {
+                            if (it.contains(OrganizationMenuBar.regexForUrlClassification)) {
+                                "organization"
+                            } else if (it.contains(ProjectMenuBar.regexForUrlClassification)) {
+                                "project"
+                            } else if (it.contains(BenchmarkCategoryEnum.regexForUrlClassification)) {
+                                "archive"
+                            } else {
+                                null
+                            }
+                        }
+                        var currentPath = "#"
+                        forEachIndexed { index: Int, pathPart: String ->
```

Comment:
Why did you replace an existing `foldIndexed`?

Reply:
When we are inside the tab, we don't need to stack the path with the parts "organization", "project: etc. And ```foldIndexed``` automatically counts, so need to rewrite it

Reply:
In cases like this please repeat what has been discussed elsewhere here too - because it might be important to keep history.

Automatically created `href`s in TopBar contain some that lead to non-existant pages, like `/organization/organizationName`. There are two possibilities to solve this:
* Create `Navigate { to ... }` in `App.kt` to redirect from those non-existent hrefs to existing ones
* Create logic in TopBar which would fix some of the links

The first option leads to a lot of new routes, because no routes had prefixes previously. For now, the second option is preferable, but we'll need to rethink all routes (only their text representation) after this PR is merged.